docs/_posts/2019-08-07-vault-target-api-apply-prep.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Vault, the Target API, and apply_prep
+---
+
+Welcome to the *first ever* Bolt developer update! This is a new experiment we're trying, to communicate more directly about what we're actively working on and what we're thinking about in the near term.
```

Comment:
Not sure the comma makes sense after `experiment we're trying`. Could drop it or make it two sentences.

Reply:
I wasn't sure either, but I figured I was overthinking it and nobody would notice. Guess not! :) I'll remove the comma.